ui/trace_window.cpp

```diff
@@ -219,6 +219,15 @@ TraceDialog::TraceDialog(ApplicationController &controller, QWidget *parent) :
     m_gfxr_capture_file_local_directory_label->hide();
     m_gfxr_capture_file_local_directory_input_box->hide();
 
+    m_pm4_capture_file_local_directory_layout = new QHBoxLayout();
```

Comment:
what is the current workflow for pm4 captures? are they saved to where executable is?

Reply:
yes, it was hardcoded as ` std::string           download_path = ".";`

Reply:
i am wondering if it is better to have the same custom path for gfxr and pm4 so that we don't have too much options in the UI. do you see any potential issue if the pm4 capture and gfxr capture uses the same folder? the pm4 capture directly from the app would have a different name anyways.

Reply:
yes, we can, but I'm not sure how to do that, currently the capture save path is only available for gfxr capture. @GrantComm maybe knows more